#Request
geometry_msgs/Point position
float64[3] orientation
bool w_orient
---
#Response
bool success